### ISSUE TITLE
Small extension to "Solve Discussion Questions: Many discussion questions are not effective for prompting"

### ISSUE THREAD

**Issue by ianb** [trigaten/Learn_Prompting#68]
In [this section](https://learnprompting.org/docs/applied_prompting/short_response#many-discussion-questions-are-not-effective-for-prompting) you have the reader theoretically rewrite the question to make it better. This itself is something you can ask the LLM to do, and I thought it might be a good example to embed in that section.

I was able to do:

```
Rephrase this as an essay description:
"The Civil War a conflict over expansion? Agree our Disagree and why?"

> "Write an essay about whether or not the Civil War was a conflict over expansion. In your essay, take a stance on the issue and provide evidence and reasoning to support your argument."
```

With a more modest prompt ("Rephrase this as a class discussion topic:") it fixed the grammar and made slight improvements, but probably not as much as desired.

(There is a typo "Agree *our* Disagree" ...  not sure if that was intentional, but it's something GPT can fix so I kind of like having it in there)

**Comment by trigaten** [comment]
Thank you! @jayo78 can you take a look?

**Comment by ianb** [comment]
There's also a typo in that section: **Expalin** the causes of the Civil War and whether expansion played a role in the conflict.

**Comment by ianb** [comment]
Oh wait, I got excited and submitted this before reading ahead! Sorry, this is already covered.
